core/src/components/footer/footer.tsx

```diff
@@ -38,6 +86,8 @@ export class Footer implements ComponentInterface {
 
           [`footer-translucent`]: translucent,
           [`footer-translucent-${mode}`]: translucent,
+
+          [`header-collapse-${collapse}`]: true,
```

Comment:
```suggestion
          [`footer-collapse-${collapse}`]: collapse !== undefined,
```
Both to fix a typo(?) and to ensure you don't get `footer-collapse-undefined` as a class, so it looks less wonky.